drivers/arm/ccn/ccn.c

```diff
@@ -490,3 +491,101 @@ int ccn_get_part0_id(uintptr_t periphbase)
 	return (int)(mmio_read_64(periphbase
 			+ MN_PERIPH_ID_0_1_OFFSET) & 0xFF);
 }
+
+/*******************************************************************************
+ * This function returns the region id corresponding to a node_id of node_type.
+ ******************************************************************************/
+static unsigned int get_region_id_for_node(node_types_t node_type,
+						unsigned int node_id)
+{
+	unsigned int mn_reg_off, region_id;
+	unsigned long long node_bitmap;
+	unsigned int loc_node_id, node_pos_in_map = 0;
+
+	assert(node_type < NUM_NODE_TYPES);
+	assert(node_id < MAX_RN_NODES);
+
+	/*
```

Comment:
The lines 508 - 516 are better placed below the switch case statement, since RNF node type is not supported. The comment can be updated for the same.

Reply:
Shifted the comments below switch case statement. Comments are updated as follows:
  /*
   * RN-I, HN-F, HN-I, SN node registers in the MN region
   * occupy contiguous 16 byte apart offsets.
   *
   * RN-F and RN-D node are not supported as 
   * none of them exposes any memory map to
   * configure any of their offset registers.
   */